packages/dev/core/src/Engines/constants.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 /** Defines the cross module used constants to avoid circular dependencies */
 export class Constants {
+    /** Sampler suffix when associated with a texture name */
+    public static readonly AUTOSAMPLERSUFFIX = "Sampler";
+    /** Flag used to disable diagnostics for WebGPU */
+    public static readonly DISABLEUA = "#define DIAGNOSTIC_OFF";

```

Comment:
I don't think `DIAGNOSTIC_OFF` is a good name, because `diagnostic(off, XXX)` is used in WebGPU to potentially disable different things (XXX). You can pass `disable_uniformity_analysis` to disable UA, but you can/could also pass other keywords to disable other things in the future. So, something like `#define DISABLE_UNIFORMITY_ANALYSIS` would probably be a better name.